ajax/libs/starability/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "starability",
+  "filename": "starability-all.min.css",
+  "version": "2.0.0",
+  "description": "Starability.css is an accessible rating system with cute animations on top. No JavaScript included.",
+  "author": "Anna Migas <amjmigas@gmail.com>",
+  "keywords": [
+    "rating",
+    "accessibility",
+    "css",
+    "sass",
+    "scss",
+    "animations"
+  ],
+  "homepage": "https://lunarlogic.github.io/starability/",
```

Comment:
The SSL certificate is un-trust, let's use `http`
Thanks

Reply:
It works for me...But I will change it